packages/styled-components/src/sheet/test/HoistedTag.test.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+
+import { VirtualTag } from '../Tag';
+import { DefaultHoistedTag } from '../HoistedTag';
+
+describe('Works like a groupedTag if there are no hoisted rules', () => {
```

Comment:
small nit: lets keep describe descriptions as names of exports or instances, instead of full descriptions like “it”